### ISSUE TITLE
training and running the trained models

### ISSUE THREAD

**Issue by deyaa97** [sunghoonhong/AirsimDRL#3]
Hello dear  sunghoonhong,
thanks alot for the interesting code :)

I am training the agent via the rddpg_per algorithm and don't know exactly how long should I train it niether how to stop the training and run the trained model.

Can you please help me with that?
Thanks

**Comment by sunghoonhong** [comment]
Hello deyaa97,

Thanks for your attention to my repository.
I'm sorry to say this, but I did tons of trials for tuning hyper-parameters.
Unfortunately, as I wrote in readme, I cannot reproduce it perfectly :/
I think you can try from 'Easy' environment first, then adjust some hyperparameters. (e.g. learning rate, action noise)
I didn't limit the training steps, i.e. no specific condition for termination.
In my case, I terminated the training when the training scores reach high enough.

If you have any question, leave to it.

Thank you
